pythonfmu/builder.py

```diff
@@ -261,13 +278,14 @@ def build_FMU(
                         temp_dest = temp_dir / file_.name
                         shutil.copytree(file_, temp_dest)
                     else:
+                        assert file_.name != script_file.name, ( # avoid the inclusion of the script in project files
+                            "It seems that the script file is included a second time in the project_files")
```

Comment:
Not sure if we want to only check the name of the file for deciding the assertion. The file name be same in different python modules.